hub/templates/hub/area.html

```diff
@@ -39,10 +39,40 @@ <h1 class="m-0">{{ area.name }}</h1>
                 </div>
             </div>
             <div class="col-lg-9 offset-xl-1">
-
+            {% if area_type == "WMC" %}
+            <div class="card flex-grow-1 dataset-card">
+                                <div class="dataset-card-header">
+                                    <h5>This is a 2010 constituency</h5>
+                                </div>
+                                <div class="card-body">
+                                    <p>At the next election, people from this constituency will be divided into <b>{{ overlap_constituencies|length|apnumber }} constituenc{% if overlap_constituencies|length_is:1 %}y{% else %}ies{% endif %}:</b></p>
+                                    <table class="table mb-0">
+                                    <tbody>
+                                        {% for overlap_constituency in overlap_constituencies %}
+                                        <tr>
+                                            <th><a href="{% url 'area' area_type=overlap_constituency.area.area_type.code name=overlap_constituency.area.name %}" class="text-decoration-none">{{ overlap_constituency.area }}</a></th>
+                                            <td class="text-muted">Covers approximately {{ overlap_constituency.pop_overlap }}% of this constituency's population, and {{ overlap_constituency.area_overlap }}% of this constituency's area.</td>
+                                        </tr>
+                                        {% endfor %}
+                                    </tbody>
+                                </table>
+                                </div>
+                            </div>
+            {% endif %}
                 <section id="mp" class="area-section">
                     <h2 class="mb-3 text-primary">MP</h2>
-                    {% include 'hub/area/_mp_data.html' with mp_data=mp %}
+                    {% if area_type == "WMC23" %}
```

Comment:
We’ll probably be replacing the "MP" section with a "Candidates" section, in the run-up to the election. But for now, I’d be tempted to just hide the MP section (and its corresponding link in the subnav) entirely, on WMC23 pages.